src/iidm/Network.cpp

```diff
@@ -174,6 +195,14 @@ const stdcxx::DateTime& Network::getCaseDate() const {
     return m_caseDate;
 }
 
+const ConnectedComponentsManager& Network::getConnectedComponentsManager() const {
+    return m_variants.get().getConnectedComponentsManager();
+}
+
+ConnectedComponentsManager& Network::getConnectedComponentsManager() {
+    return m_variants.get().getConnectedComponentsManager();
+}
```

Comment:
To be discuss: from the public API the connected components are read-only data (same remark for synchronous components)